src/main/java/com/twilio/oai/TwilioNodeGenerator.java

```diff
@@ -213,9 +213,15 @@ public OperationsMap postProcessOperationsWithModels(final OperationsMap objs, L
                     resource.put("responseModel", responseModel);
                     responseModel
                         .getVars()
-                        .forEach(variable -> directoryStructureService.addModel(models,
-                                                                                variable.complexType,
-                                                                                variable.dataType));
+                        .forEach(variable -> {
+                            directoryStructureService.addModel(models,
+                                    variable.complexType,
+                                    variable.dataType);
+                            if (variable.baseName.equals("uri")) {
+                                responseModel.vendorExtensions.put("x-omit-uri", true);
```

Comment:
I don't think this is the proper fix. `Page.TwilioResponsePayload` is what we're extending and causing the conflict, but the properties of a resource are not at the top level of the payload. They're nested under the _record key_. E.g., https://github.com/twilio/twilio-oai/blob/13f971db9f33d269d96a5c1fe3cedc651d403c70/spec/yaml/twilio_accounts_v1.yaml#L227-L252

Reply:
So...should `uri` be removed from response model types entirely? Or would it be easier to just make `uri`s optional and nullable by default in the `Page.TwilioResponsePayload` type

Reply:
I think the overall usage of `Page.TwilioResponsePayload` needs to be refactored as it's incorrect (this is true of the main branch as well). In some cases we need to extend it (v2010) and others need to wrap it (non-v2010).